tests/models/test_model.py

```diff
@@ -91,6 +93,38 @@ def test_model_log():
         path = os.path.join(local_path, loaded_model.saved_input_example_info["artifact_path"])
         x = _dataframe_from_json(path)
         assert x.to_dict(orient="records")[0] == input_example
+        assert not hasattr(loaded_model, "databricks_runtime")
+
+
+def test_model_log_with_databricks_runtime():
+    dbr = "8.3.x-snapshot-gpu-ml-scala2.12"
+    with TempDir(chdr=True) as tmp, mock.patch(
+        "mlflow.models.model.get_databricks_runtime", return_value=dbr
+    ):
+        experiment_id = mlflow.create_experiment("test")
+        sig = ModelSignature(
```

Comment:
Just as a general bit of refactoring, we may want to consider factoring out some of this model logging boilerplate into a helper function and using it in all the other functions in this test too.